x-pack/plugins/viz_editor/public/pseudo_plugins/indexpattern_datasource_plugin/to_expression.ts

```diff
@@ -41,6 +41,7 @@ function queryToEsAggsConfigs(query: Query): any {
           type: 'sum',
         };
       case 'terms':
+      case 'column':
```

Comment:
So column behaves as a terms agg if there are other aggs and as a "field selection" if its a "raw documents" query? I'm not sure whether we want to allow this, maybe it makes more sense to explicitly switch from columns to terms in the editor if you want to do this switch?

Reply:
Removing